packages/components/src/resizable-box/style.scss

```diff
@@ -37,16 +59,42 @@
 
 // Make horizontal side-handles full width
 .components-resizable-box__side-handle.components-resizable-box__handle-top,
-.components-resizable-box__side-handle.components-resizable-box__handle-bottom {
+.components-resizable-box__side-handle.components-resizable-box__handle-bottom,
+.components-resizable-box__side-handle.components-resizable-box__handle-top::before,
+.components-resizable-box__side-handle.components-resizable-box__handle-bottom::before {
 	width: 100%;
 	left: 0;
+	border-left: 0;
+	border-right: 0;
 }
 
 // Make vertical side-handles full height
 .components-resizable-box__side-handle.components-resizable-box__handle-left,
-.components-resizable-box__side-handle.components-resizable-box__handle-right {
+.components-resizable-box__side-handle.components-resizable-box__handle-right,
+.components-resizable-box__side-handle.components-resizable-box__handle-left::before,
+.components-resizable-box__side-handle.components-resizable-box__handle-right::before {
 	height: 100%;
 	top: 0;
+	border-top: 0;
+	border-bottom: 0;
+}
+
+// Hide side handle line by default
+.components-resizable-box__side-handle.components-resizable-box__handle-top::before,
+.components-resizable-box__side-handle.components-resizable-box__handle-bottom::before {
+	transform: scaleX(0);
```

Comment:
A couple animation notes: Just noting that scale is one of the less performant animations. I don't think it's a huge issue here since we're only animating borders basically. Shouldn't have a huge impact. 

In any case though, we should move these into an `animation` property and combine it with `opacity` animation as well so this animation relates a bit more to some of the other animations we have happening in the UI. Then we can also loop in the [`@reduce-motion` mixin](https://github.com/WordPress/gutenberg/blob/a76a702501bc2f2ab4b9eaef8fadb8e93f24436b/assets/stylesheets/_mixins.scss#L336-L344) for users that want to opt out of the animation as well. 

@marekhrabe I have that change ready locally — do you mind if I push it to this branch?

Reply:
> I have that change ready locally — do you mind if I push it to this branch?

You rock! Please do 🤘